cpp/ql/src/Architecture/General Class-Level Information/InheritanceDepthDistribution.qhelp

```diff
@@ -5,44 +5,34 @@
 
 
 <overview>
-<p>This query shows the distribution of inheritance depth across all types, i.e. classes. Library types are ignored.</p>
+<p>This query shows the distribution of inheritance depth across all types, that is, classes. Library types are ignored.</p>
 
 <p>The result of this query is a line graph showing, for each number <i>n</i>, how many types have an inheritance depth of <i>n</i>, where
 the inheritance depth of a type is the length of a longest path in the inheritance hierarchy from top class to the type.</p>
-
-<p>When hovering the mouse pointer over a specific depth value, the number of types having this inheritance depth is displayed.</p>
-
 </overview>
-<section title="How to Address the Query Results">
+
+<recommendation>
 <p>The depth of a type is an indication of how deeply nested a type is in a given design. 
 Very deep types can be an indication of over-engineering, whereas a system with predominantly shallow types 
 may not be exploiting object-orientation to the full.</p>
+</recommendation>
 
-
-
-
-
-</section>
 <references>
 <li>
 Shyam R. Chidamber and Chris F. Kemerer.
 <a href="http://www.pitt.edu/~ckemerer/CK%20research%20papers/MetricForOOD_ChidamberKemerer94.pdf">A Metrics Suite for Object Oriented Design
 </a>.
 IEEE Transactions on Software Engineering,
-20(6), pages 476-493, June 1994.
-
-
+20(6), pages 476-493, June 1994.</li>
 
+<li>
 <a href="http://www.dmst.aueb.gr/dds/index.en.html">Diomides D. Spinnelis</a>.
```

Comment:
It isn't clear to me which part of the above page I should be looking at.  I don't doubt there's something relevant on there somewhere.

Reply:
Oh, he's the author of the book below.  As long as this is more obvious in the rendered page.

Reply:
The changes in this PR just fix the obvious formatting issues with the references. Having looked at the references, I'm not sure that any of them say anything useful.

Would any of these be better?

* http://people.scs.carleton.ca/~jeanpier//sharedF14/T1/extra%20stuff/about%20metrics/Chidamber%20&%20Kemerer%20object-oriented%20metrics%20suite.pdf
* http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.159.2229&rep=rep1&type=pdf
* https://www.researchgate.net/publication/266139603_Object-Oriented_Inheritance_Metrics_in_the_Context_of_Cognitive_Complexity

Reply:
I like the first two, I'm not completely sure about the third one.

Reply:
Ah - it turns out that the first one is copied from the help for an application http://www.aivosto.com.  Will use the second one.